### ISSUE TITLE
Color contrast not appropraite for json attributes in QuiteLight theme

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#56791]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1)Navigate to Settings-->Select color theme -->Select Quiet light.
2)Check color contrast ratio for the Explorer content on the screen.

**Actual:**
Color contrast ratio is less than 4.5:1 for the black colored content when focused.
**Similar issue is observed in following themes:**
Abyss
Kimbie dark
Monokai Dimmed
High contrast
Install additional color theme

**Expected:**
Color contrast ratio should be more than 4.5:1 for the content on a screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
](url)
**User Impact:**
The low vision/visually impaired users will not be able to view the content properly.

**MAS Reference:**
MAS 1.4.3 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
](url)
**Attachment For Reference:**
![22850_a11y_vscode_colortheme_colorcontrast](https://user-images.githubusercontent.com/39204909/44325481-8ccad780-a476-11e8-988f-ea46fef29bbf.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by misolori** [comment]
Per our agreement, we will not be fixing color contrast issues with third party themes.

cc @auchenberg

**Comment by isidorn** [comment]
@misolori just a note that we are the authors of this theme (actually I am the original author)
Does this mean that we only fix for the Light+ and Dark+?

**Comment by misolori** [comment]
@isidorn we're supposed to fix any issues for themes that we own and ship with out of the box (except for third party themes we don't own). I didn't know we own QuietLight so I'll re-open.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Checked and verified in below version i.e 1.27.0-Insiders.

The issue still exist in Quite Light theme.

Similar issue is observed in following themes:
1)Solarized light:All the text in the page fail in color contrast ratio i.e 4.5:1
2)Light visual studio: Black coloured text fail in color contrast ratio  i.e 4.5:1

**Comment by misolori** [comment]
@AccessibilityTestingTeam-TCS please see #56794 for the unused variable color issues
